src/containers/TokenPageContainer/TokenPageContainer.jsx

```diff
@@ -593,7 +593,7 @@ const TokenPageContainer = () => {
 
       try {
         const tokenData = offerData?.nftContract?.tokens?.find(
-          (token) => Number(token?.tokenId) === Number(tokenId)
+          (token) => BigInt(token?.tokenId) === BigInt(tokenId)
```

Comment:
Are we sure `token.tokenId` & `tokenId` exist ? 
You may get "TypeError: Cannot convert undefined to a BigInt"

FYI
```
Number(undefined) => NaN
BigInt(undefined) => Type Error 
```